config/database.php

```diff
@@ -54,6 +54,13 @@
             'engine' => null,
         ],
 
+        'mariadb' => [
+            'driver' => 'mariadb',
+            'url' => env('DB_URL'),
+            'host' => env('DB_HOST', '127.0.0.1'),
+            'port' => env('DB_PORT', '3306'),
+        ],
+
```

Comment:
This shouldn't be necessary since we use MySQL

Reply:
It's straight out of the upgrade guide... it just adds a driver for mariadb to the list of other drivers. Since I'm considering changing providers and maria is compatible with mysql, it made sense to add this. https://laravel.com/docs/11.x/upgrade#dedicated-mariadb-driver